staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec_test.go

```diff
@@ -450,17 +451,29 @@ func TestRefreshCreds(t *testing.T) {
 			}`,
 			wantErr: true,
 		},
+		{
+			name: "install-hint",
+			config: api.ExecConfig{
+				APIVersion:  "client.authentication.k8s.io/v1beta1",
+				Command:     "does not exist",
+				InstallHint: "some install hint with \x1b[1mcontrol chars\x1b[0m\nand a newline",
+			},
+			wantErr:       true,
+			wantErrSubstr: "some install hint with U+001B[1mcontrol charsU+001B[0m\nand a newline",
+		},
```

Comment:
Are we testing cases where the binary exists but errors on execution?  Basically we need to test all the error outputs from `cmd.Run()`.

Reply:
Added another test in https://github.com/kubernetes/kubernetes/pull/91305/commits/e53aca0c6f8c9a2c82abafd2e7baf6bd0b1c894e to try to address this.